.babelrc

```diff
@@ -1,7 +1,38 @@
 {
   "presets": [
-    ["jason", {"debug": false}],
     "flow"
   ],
-  "sourceMaps": "inline"
+  "env": {
+    "development": {
+      "presets": [
+        [
+          "env", {
+            "targets": {
+              "browsers": ["last 2 versions", "safari >= 7"]
+            }
+          }
+        ]
+      ],
+      "plugins": [
+        "transform-class-properties",
+        "transform-react-jsx",
+      ]
+    },
+    "production": {
+      "presets": [
+        [
+          "env", {
+            "modules": false,
+            "targets": {
+              "browsers": ["last 2 versions", "safari >= 7"]
+            }
+          }
+        ]
+      ],
+      "plugins": [
+        "transform-class-properties",
+        "external-helpers"
```

Comment:
what does this do?

Reply:
Sorry I don't follow. What are you trying to ask?

Reply:
the `external-helpers` plugin, it looks like it extracts the babel helpers out of the code, but we don't have anything that provides them back, e.g. https://babeljs.io/docs/plugins/external-helpers/#getting-the-external-helpers

My question tho is more, what is the rational for adding this in the first place?

Reply:
Hope this helps: https://github.com/rollup/rollup-plugin-babel#helpers

Reply:
great should we turn that feature on then?

Reply:
Sorry I dont follow again :) What feature? External helpers was a way to remove duplicated babel helpers across the bundle.

Reply:
Do you find duped helper code in the bundle?

Reply:
the documentation for rollup babel says you need to enable that feature with `externalHelpers: true`

Reply:
> Finally, if you do not wish the babel helpers to be included in your bundle at all (but instead reference the global babelHelpers object), you may set the externalHelpers option to true

Babel's docs say by default babelHelpers is created as a global by default, but rollup-plugin-babel it will combine it and place it at the top

> By default, those helpers will be inserted at the top of the file being transformed, which can lead to duplication. It's therefore recommended that you use the external-helpers plugin, which is automatically included in the es2015-rollup preset. Rollup will combine the helpers in a single block at the top of your bundle.

externalHelpers: true is to be used when these helpers are referenced from babelHelpers global. I just grepped the source and couldn't find any (build isn't configured to do that anyway. So it makes sense.)

Did you find duplicated helper code?

Reply:
i wasn't worried about duplicated code, but the plugin removing the helper references but not including them back at the top. I see tho it seems to do that automatically with rollup so nothing else to do i guess